translator/toenvconfig/toEnvConfig.go

```diff
@@ -36,6 +37,9 @@ func ToEnvConfig(jsonConfigValue map[string]interface{}) []byte {
 		if isDebug, ok := agentMap[debugKey].(bool); ok && isDebug {
 			envVars[envconfig.CWAGENT_LOG_LEVEL] = "DEBUG"
 		}
+		if awsSDKLogLevel, ok := agentMap[awsSDKLogLevelKey].(string); ok {
```

Comment:
Just remembered, don't you need to update translator/config/schema.go with this key as well?

Reply:
Fixed.